requirements/development.in

```diff
@@ -17,26 +17,4 @@
 # under the License.
 #
 -r base.in
--e .[bigquery,cors,druid,gevent,gsheets,hive,mysql,playwright,postgres,presto,prophet,trino,thumbnails]
-
-docker
-flask-testing
-freezegun
-greenlet>=2.0.2
-grpcio>=1.55.3
-ipython
-openapi-spec-validator
-parameterized
-pip-compile-multi
-pre-commit
-progress>=1.5,<2
-pyfakefs
-pyinstrument>=4.0.2,<5
-pylint
-pytest
-pytest-cov
-pytest-mock
-python-ldap>=3.4.4
-sqloxide
-statsd
-tox
+-e .[development,bigquery,cors,druid,gevent,gsheets,hive,mysql,playwright,postgres,presto,prophet,trino,thumbnails]
```

Comment:
note that I added `development` here, which is referencing a group that contains the libs referenced in the file bellow, now in pyproject.toml